src/chains/filecoin/filecoin/package.json

```diff
@@ -27,9 +31,10 @@
     "docs.build": "rm -rf ./lib/docs ./lib/api.json && npm run docs.typedoc",
     "docs.typedoc": "typedoc --options ./typedoc.json --readme ./README.md --out ../../../docs/typedoc --json ../../../docs/typedoc/api.json src/api.ts",
     "docs.preview": "ws --open --port 3012 --directory ../../../",
-    "tsc": "ttsc",
+    "tsc": "ts-node ../../../../scripts/lerna-run.ts ttsc --project _PACKAGEDIR_/tsconfig.json",
+    "tsc.declarations": "ts-node ../../../../scripts/lerna-run.ts ttsc --project _PACKAGEDIR_/tsconfig.declarations.json",
     "test": "nyc --reporter lcov npm run mocha",
-    "mocha": "cross-env TS_NODE_COMPILER=ttypescript TS_NODE_FILES=true mocha --exit --check-leaks --throw-deprecation --trace-warnings --require ts-node/register --timeout 10000 'tests/**/*.test.ts'"
+    "mocha": "cross-env TS_NODE_COMPILER=ttypescript TS_NODE_FILES=true ts-node ../../../../scripts/lerna-mocha.ts --require ts-node/register '_PACKAGEDIR_/tests/**/*.test.ts'"
```

Comment:
All of the `filecoin` branches are currently based off of #757 as it helps my development process. This adheres to that if it is accepted. It will be changed if #757 is rejected.